test/Datadog.Trace.Tests/DogStatsDTests.cs

```diff
@@ -62,9 +69,33 @@ public void Send_metrics_when_enabled()
                 s => s.Add<Statsd.Counting, int>(TracerMetricNames.Api.Requests, 1, 1, null),
                 Times.Once());
 
-            statsd.Verify(
```

Comment:
I'm also open to suggestions on how we could make changes so we always get a successful connection to the agent, instead of testing for either success or failure connecting